reconcile/dashdotdb_cso.py

```diff
@@ -6,18 +6,34 @@
     LOG,
     DashdotdbBase,
 )
+from reconcile.typed_queries.app_interface_vault_settings import (
+    get_app_interface_vault_settings,
+)
 from reconcile.utils.oc import (
     OC_Map,
     StatusCodeError,
 )
+from reconcile.utils.secret_reader import (
+    SecretReaderBase,
+    create_secret_reader,
+)
 
 QONTRACT_INTEGRATION = "dashdotdb-cso"
 LOGMARKER = "DDDB_CSO:"
 
 
 class DashdotdbCSO(DashdotdbBase):
-    def __init__(self, dry_run, thread_pool_size):
-        super().__init__(dry_run, thread_pool_size, LOGMARKER, "imagemanifestvuln")
+    def __init__(
+        self, dry_run: bool, thread_pool_size: int, secret_reader: SecretReaderBase
+    ):
```

Comment:
It would be great to see type hints in this module and the other ones too :)

Reply:
You are right. Added more type hints here.